cmd/fossa/flags/flags.go

```diff
@@ -49,7 +49,7 @@ var (
 	Endpoint        = "endpoint"
 	EndpointF       = cli.StringFlag{Name: Short(Endpoint), Usage: "the FOSSA server endpoint (default: 'https://app.fossa.com')"}
 	Title           = "title"
-	TitleF          = cli.StringFlag{Name: Short(Title), Usage: "the title of the FOSSA project. (default: the project name)"}
+	TitleF          = cli.StringFlag{Name: Short(Title), Usage: "the title of the FOSSA project, valid ONLY on project creation (default: the project name)"}
```

Comment:
Be more precise than "valid on". What does it mean for a title to be _valid?_

I would suggest "the title of the FOSSA project (applies only to new projects)".

Reply:
I like this change! Thanks for the rec.